src/layouts/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Helmet from "react-helmet";
+import PropTypes from "prop-types";
+import fontAwesome from "@fortawesome/fontawesome";
+import brands from "@fortawesome/fontawesome-free-brands";
+import solid from "@fortawesome/fontawesome-free-solid";
+import { ThemeProvider } from "@hackclub/design-system";
+
+// Easier reference to Font Awesome icons from components
+fontAwesome.library.add(brands, solid);
+
+const DefaultLayout = ({ children }) => (
+    <ThemeProvider>
+        <Helmet title="Finder — Hack Club" />
```

Comment:
**if you only take one suggestion of this list, please make it this** (otherwise i'll create an issue for you, lol). **add social metadata!** look at the `pages` on `site` and copy their `Helmet` code. but we need all those tags, and keyword-rich page descriptions. #GoogleJuice